app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2525,9 +2528,16 @@ class BrowserTabFragment :
     }
 
     private fun launchAutofillManagementScreen() {
+        val privacyProtectionStatus = when (viewModel.privacyShieldViewState.value?.privacyShield) {
+            PROTECTED -> PrivacyProtectionStatus.Enabled
```

Comment:
I'm unsure this is the logic you want. I would say is wrong to rely on the privacy shield to decide if protections are enabled/disabled. http site is unprotected, but protections are enabled.

Reply:
As discussed offline, changed how this value is determined. Thanks!